fe/fe-core/src/main/java/org/apache/doris/catalog/ColocateTableIndex.java

```diff
@@ -176,7 +176,7 @@ public static boolean isGlobalGroupName(String groupName) {
     private Multimap<GroupId, Long> group2Tables = ArrayListMultimap.create();
     // table_id -> group_id
     @SerializedName(value = "table2Group")
-    private Map<Long, GroupId> table2Group = Maps.newHashMap();
+    private Map<Long, GroupId> table2Group = Maps.newConcurrentMap();
```

Comment:
It is serialized via gson, we should handle gson deserialization, otherwise, after gson deserialization, table2Group is not a concurrent hash map.

Reply:
ColocateTableIndex not use gson serialization，  ColocateTableIndex still use the old way of serialization
@LiBinfeng-01  add this 
`@SerializedName(value = "table2Group")
`
just for planner minidump.

So there's no problem here